components/brave_wallet_ui/panel/container.tsx

```diff
@@ -198,14 +193,6 @@ function Container() {
     BraveWallet.NetworkInfo | undefined
   >(undefined)
 
-  const unlockWallet = (password: string) => {
-    dispatch(WalletActions.unlockWallet({ password }))
-  }
-
-  const onRestore = () => {
-    dispatch(WalletPanelActions.expandRestoreWallet())
```

Comment:
We don't need `expandRestoreWallet`` action anymore
@Douglashdaniel fyi

Reply:
Yes looks like you can remove!

Reply:
Removed in https://github.com/brave/brave-core/pull/20898/commits/51bc817662003dcb52d2f326b94ad2073066a55d